__tests__/app.test.js

```diff
@@ -85,3 +86,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+//5
+describe("/api/articles", () => {
+  test("GET - status 200 - responds list of articles sorted date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { allArticles } = response.body;
+
+        expect(allArticles.length > 0).toBe(true);
+        allArticles.forEach((article) => {
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+
+          expect(typeof article.body).toBe("undefined");
+        });
+
+        expect(allArticles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
```

Comment:
Nice test to check that data is sorted by date, i'd probably encourage you put this into a separate test block but it's not the end of the world if not